src/test/java/com/iexec/worker/config/SchedulerConfigurationTests.java

```diff
@@ -24,11 +24,10 @@
 import org.springframework.beans.factory.BeanCreationException;
 import org.springframework.boot.context.annotation.UserConfigurations;
 import org.springframework.boot.test.context.runner.ApplicationContextRunner;
-import org.springframework.boot.test.system.CapturedOutput;
```

Comment:
By removing this import, it should mean the `@ExtendWith(OutputCaptureExtension.class)` can be removed as well